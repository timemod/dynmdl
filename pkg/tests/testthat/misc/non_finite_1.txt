No convergence after 1 iterations
Non-finite values in residuals for the following equations:
eq 2 
-Inf 
