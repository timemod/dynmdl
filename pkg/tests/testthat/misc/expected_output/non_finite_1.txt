Function value contains non-finite values (starting at index=2)
Non-finite values in residuals for the following equations:
eq 2 
-Inf 
